sdk/docker-compose.yml

```diff
@@ -2,6 +2,18 @@ version: "3"
 
 services:
 
+  tinyproxy:
+    image: dannydirect/tinyproxy:latest
```

Comment:
Should we use `dannydirect`?

Reply:
Its only for the tests in the travis build, I think its ok